app/controllers/AccountManagement.scala

```diff
@@ -167,16 +163,18 @@ object ManageWeekly extends ContextLogging {
   import play.api.mvc.Results._
 
   // this sequencing concatenates errors if any, otherwise aggregates rights
-  def sequence[A](list: List[\/[String, A]]): \/[String, List[A]] = {
-    val errors = list collect {
-      case -\/(x) => x
+  def sequence[A](list: List[\/[NonEmptyList[String], A]]): \/[NonEmptyList[String], List[A]] = {
```

Comment:
the change here is to make it concatenate the errors into a list instead of a string, which seems neater and helped the types